Jacksonmcguire/intention-timer
# Feature/form

## Is this a fix or a feature?
Feature
## What is the change?
Adds feature to validate number input, prevents user from being able to input anything but a number in minutes and seconds input field. 
## What does it fix?
N/A
## Where should the reviewer start?
The reviewer should review line 27-38. 
## How should this be tested?
The reviewer should try to enter every character in both the minutes and seconds input boxes and confirm only numbers can be passed.
